tox.ini

```diff
@@ -163,7 +166,8 @@ commands_pre=
     pip install -r {envtmpdir}/requirements.txt
     pip freeze
 commands=
-    pytest specutils
+    pytest specutils \
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.types:504"
```

Comment:
This will be fixed by the above request on the entry point.

Reply:
This is still required because specutils contains a subclass of CustomType.

Reply:
Really we should not be suppressing warnings in our downstream. This should then be an expected failure.

Reply:
I just opened astropy/specutils#1011 to track the expected failure in the downstream.

Reply:
To clarify, do you want this line removed and the downstream specutils job to fail?

Reply:
Yes. The same for the astropy job. We are expecting those jobs to fail.

Reply:
I do not think we should allow these to fail when we have a simple way to keep them running to find errors for the other changes we are making. If we do not include these warning filters they will fail during test collection and run no tests. This means that any change we make in asdf that might break these downstream packages would go unnoticed (instead of appearing in our CI with the addition of these warning filters).

Reply:
If you want to go this route then you should track the "fixes" in the downstream packages and when they are merged, removing them from ASDF.

Reply:
That sounds good to me.